### ISSUE TITLE
download still fail after adding new device.wvd

### ISSUE THREAD

**Issue by phwaleed2024** [glomatico/spotify-web-downloader#82]
i added anew and working device.wvd file and i still get the error 
please do not close the issue untill i fix this problem

**Comment by youngoris** [comment]
Me neither. I'm encountering the same issue even after trying with a new device.wvd file. Here are the details of my setup:

AVD: Pixel 6 Pro API 34 Android 14.0 ("UpsideDownCake")
SWD: 1.8
OS: macOS 14.6
Hopefully, someone can help us resolve this or provide a workaround.

**Comment by phwaleed2024** [comment]
i wonder the developer sure of it is working and the problem is from us !!!
i did all he said and it still not working even it worked before 3 days ago
i hope he find asolution or try himself the script in this moment and give us ascreenshot thanksfully of work

**Comment by chu23465** [comment]
I checked 13 different cdms mostly from emulators, 3 from actual devices. All were blocked by spotify. If anybody found a working one, please do share.

**Comment by vijaykrpp** [comment]
> I checked 13 different WVDs mostly from emulators, 3 from actual devices. All were blocked by spotify. If anybody found a working one, please do share.

Did your any wvd worked? I tried from 3 AVDs and all never worked for a single time.

**Comment by glomatico** [comment]
Duplicate of #70
